curriculum-crawler.iml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Try not to import the iml files, other devs may have their own IDE settings and you don't want to enforce your settings on them.